recipes/aab/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "aab" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aab-{{ version }}.tar.gz
+  sha256: 1be9cd6dceac28c570df268a2aa20c1935a59043c88c9f4ce88a6d208e007d09
+
+build:
+  skip: true  # [py>=400]
+  entry_points:
+    - aab=aab.cli:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six
+    - jsonschema
+    - whichcraft
+    - pathlib  # [py<34]
```

Comment:
You can make this package noarch by limiting to python>=3.6

Reply:
I think it does with the latest changes